grassroots-frontend/src/routes/Search.tsx

```diff
@@ -30,15 +30,19 @@ function Search(): JSX.Element {
 
   const [rowsToSkip, setRowsToSkip] = useState<number>(0);
 
-  const searchParams: PaginatedContactSearchRequestDTO = {
-    contact: plainToInstance(ContactSearchRequestDTO, form.watch()),
+  const searchParams = PaginatedContactSearchRequestDTO.from({
+    contact: form.watch(),
```

Comment:
I REALLY like how this approach makes it clear when it needs to be used compared to "plain to instance" which lacks context of why it's need, so make it hard to know when it's needed.